### DISCUSSION TITLE
RavenDB support

### DISCUSSION THREAD

**Discussion by adambajguz** [CommunityToolkit/Aspire#104] [Ideas]
Hi, are there any plans of adding [RavenDB](https://github.com/ravendb/ravendb) support?

https://github.com/dotnet/aspire/discussions/2119
https://github.com/ravendb/ravendb/discussions/19464

**Comment by aaronpowell** [comment]
I'll be honest, I haven't used RavenDB so I'm unsure what would be required to get an integration created for it across hosting and I'd assume client.

If someone wanted to submit a PR with the integrations, we can review it for inclusion in the toolkit, but I'd also want to hear from the RavenDB folks on whether it's something they are considering producing themselves. I see you've created a discussion on their GitHub and a quick scan of their issues produces https://github.com/ravendb/ravendb/issues/18706 which seems to suggest that it might be something they want to ship themselves.

**Reply by aaronpowell** [reply]
I'm going to close this as it's being tracked by the RavenDB product team - https://issues.hibernatingrhinos.com/issue/RavenDB-23016/Add-.NET-Aspire-Integration-for-RavenDB
